spec/fixtures/vcr/_page_count_returns_total_count.yml

```diff
@@ -2,7 +2,7 @@
 http_interactions:
 - request:
     method: get
-    uri: https://pdc.host.dev.eng.pek2.redhat.com/rest_api/v1/arches/?page=2
+    uri: https://pdc.test.com/rest_api/v1/arches/?page=2
```

Comment:
Dear @xiangge, thanks for the update. Perhaps instead of `pdc.test.com` why not use the URI to Fedoras PDC. That's public knowledge and a deployed server. If the problem is that it might be confusing for other contributors I'd suggest to use an example domain like [.test](https://en.wikipedia.org/wiki/.test).

It would be good to keep just the URL changes in a separate patch as well.